src/libs/Localize/index.js

```diff
@@ -12,6 +12,19 @@ import BaseLocaleListener from './LocaleListener/BaseLocaleListener';
 // Listener when an update in Onyx happens so we use the updated locale when translating/localizing items.
 LocaleListener.connect();
 
+const CONJUNCTION_LIST_FORMATS_FOR_LOCALES = _.reduce(CONST.LOCALES, (memo, locale) => {
+    // This is not a supported locale, so we'll use ES_ES instead
+    if (locale === CONST.LOCALES.ES_ES_ONFIDO) {
+        // eslint-disable-next-line no-param-reassign
+        memo[locale] = new Intl.ListFormat(CONST.LOCALES.ES_ES, {style: 'long', type: 'conjunction'});
```

Comment:
This broke native. Intl is a Web API. `Intl.ListFormat` need to be poly-filled. https://formatjs.io/docs/polyfills/intl-listformat

Reply:
Updated the IntlPolyfill